CONTRIBUTOR-LADDER.md

```diff
@@ -150,3 +150,11 @@ Inactive members are defined as members of the Cilium org with **no** contributi
 
 **Note:** Devstats does not take into account non-code contributions. If a non-code contributing member is accidentally removed this way, they may open an issue to quickly be re-instated.
 
+## General guidance on making contributions and expectations
```

Comment:
Thanks for putting this together Liz. I think that this guidance could be useful for community members and I agree . Having read Nadia Eghbal's book _Working in Public: The Making and Maintenance of Open Source Software_, I think this reflects the dynamics in play in an open community.

I do wonder a little about a couple of things:
- Whether placing the guidance in this location will reach the target contributor audience better than an alternative location
- Whether this guidance provides sufficient encouragement to reassure potential contributors about how important their contributions are; for instance, regularly contributing to review can help to share the load and improve PR response time. Ideally if this statement is saying "we try to facilitate community contributions, but we don't have all the time in the world" and contributors will feel empowered to respond with "how can I help?" and not "oh okay I won't contribute then".

Reply:
> Whether placing the guidance in this location will reach the target contributor audience better than an alternative location

The other option I can think of would be the [Contributor Guide](https://docs.cilium.io/en/stable/contributing/development/). I'm imagining that guide will eventually [move into the community repo](https://github.com/cilium/community/issues/78) along with the governance-related sections of the docs, so that it all eventually lands in one place. I will confess that I proposed adding this text here in the meantime, because it's easier than changing the docs, but maybe that is too much laziness on my part :-D 

> contributors will feel empowered to respond with "how can I help?" and not "oh okay I won't contribute then"

Very good point. I made some text edits to attempt to address this, wdyt?